### DISCUSSION TITLE
Quota Limits

### DISCUSSION THREAD

**Discussion by joshuatownsend** [mtalcott/google-photos-deduper#15] [Q&A] [answered]
I've started experimenting with this project but hit the Photos API quota limits pretty quickly. Are there any methods to batch process (by date, album, etc.), or to pause after X number of requests to allow for quota reset?

**Comment by mtalcott** [comment, marked as answer]
Hey @joshuatownsend, I'm working on some changes that will improve the handling of daily quota, should have them up in the next few days

**Reply by joshuatownsend** [reply]
Awesome - happy to help with testing!

**Reply by mtalcott** [reply]
Great! I pushed some changes up: https://github.com/mtalcott/google-photos-deduper/pull/16. The task will now fail once quota is hit, but progress is saved and it can be restarted the next day.

**Comment by karbassi** [comment]
Hi @mtalcott, thanks for this project. 

I hit my quota pretty quickly. Is there any other ideas or fixes?
